cloud-config/nightly/cs_config_manager_config/oauth2-agents.json

```diff
@@ -2,45 +2,93 @@
   "alpha": {
     "IdentityGatewayAgent": [
       {
-        "id": "ig-agent"
+        "id": [
+          "ig-agent"
+        ],
+        "overrides": {
+          "userpassword": "&{esv.sapig.ob.ig.agent.password}"
```

Comment:
thought we needed to use the $ notation?

Reply:
No we are saying we want no matter the value in the cloud, we want to override it with this, so it matches what is in the templates we have